js/about/adblock.js

```diff
@@ -14,11 +14,11 @@ const SwitchControl = require('../../app/renderer/components/common/switchContro
 const {DefaultTextArea} = require('../../app/renderer/components/common/textbox')
 
 const {StyleSheet, css} = require('aphrodite/no-important')
+const globalStyles = require('../../app/renderer/components/styles/global')
 
 const ipc = window.chrome.ipcRenderer
 
 // Stylesheets
-require('../../less/switchControls.less')
```

Comment:
This is duping to https://github.com/brave/browser-laptop/pull/10881/files#diff-b79e96c438a5b97dc31f04f68d4817a7R7